src/task.js

```diff
@@ -7,7 +7,7 @@ function Task(
   producer: Producer,
   topic: string,
   subscribeCallback: Callback) {
-  const subscribe = (payload: any) => subscribeCallback(payload);
+  const subscribe = (payload: any) => Promise.resolve(subscribeCallback(payload));
```

Comment:
should we check like [this](https://github.com/tc39/ecmascript-asyncawait/issues/78#issuecomment-164408675)?